source/connect.txt

```diff
@@ -152,40 +239,42 @@ To rename a connection to a deployment in |vsce|:
 
 .. _vsce-disconnect-task:
 
-Disconnect from a Deployment
+Disconnect from a Connection
 ----------------------------
 
 |vsce| provides two methods to disconnect from a deployment. You can:
 
-- Disconnect with the Command Palette, or
+- Disconnect with the Command Palette.
 - Disconnect from the :guilabel:`MongoDB` view in the 
   :guilabel:`Activity Bar`.
 
 .. tabs::
 
-   .. tab:: Disconnect with the Command Palette
-      :tabid: command-palette
-
-      .. include:: /includes/steps/disconnect-command-palette.rst
-
-   .. tab:: Disconnect from the MongoDB View
+   .. tab:: With the MongoDB View
```

Comment:
*Note* - Rearrange all tasks to show the `MongoDB View` as the first tab since it is the easier of the two options.